CHANGELOG.adoc

```diff
@@ -15,6 +15,10 @@
 
 https://github.com/oxidecomputer/dropshot/compare/v0.10.1\...HEAD[Full list of commits]
 
+=== Breaking Changes
+
+* https://github.com/oxidecomputer/dropshot/pull/1028[#1028] Upgraded to hyper v1.0. This means types from the `http` must come from v1.x. Dropshot no longer expects requests and responses to use the `hyper::Body` type, but rather `dropshot::Body`. It behaves very similar.
```

Comment:
I'm looking for more specificity here, like in the examples I linked to.  But I can take a swing at this part and just push to this PR, if that works?

Reply:
I hope you don't mind that I just went ahead and did that.  (I'm surprised GitHub let me!)